packages-ts/gauntlet-terra-contracts/src/commands/contracts/migration/index.ts

```diff
@@ -0,0 +1,80 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgMigrateContract } from '@terra-money/terra.js'
+
+type CommandInput<MsgMigrate> = {
+  contract: string
+  newCodeId: number
+  migrationMsg: MsgMigrate
```

Comment:
nit: terra.js has a consistent `Msg` prefix for all messages, so let's also try to be consistent here and start with the `msg/Msg` prefix for all vars, members, and types.

```suggestion
  msgMigrate: MsgMigrate
```